src/app/components/transactionsRequests/ContractCallRequest.tsx

```diff
@@ -147,7 +147,7 @@ export default function ContractCallRequest(props: ContractCallRequestProps) {
   const getFunctionArgs = (): Array<ArgToView> => {
     const args: Array<ArgToView> = [];
     const { funcArgs } = extractFromPayload(request);
-    funcMetaData?.args?.map((arg: Args, index: number) => {
+    funcMetaData?.args?.forEach((arg: Args, index: number) => {
```

Comment:
hey, thanks for fixing these eslint errors, but in the spirit of reducing QA test scope for this PR, let's not make functional changes like this without having time to really test the implications.